tracer/test/Datadog.Trace.Tools.dd_dotnet.ArtifactTests/CreatedumpTests.cs

```diff
@@ -128,33 +135,39 @@ public async Task BashScript()
 
         helper.StandardOutput.Should().Contain(CreatedumpExpectedOutput);
     }
+#endif
 
     [SkippableTheory]
     [InlineData(true)]
     [InlineData(false)]
     public async Task DoNothingIfNotEnabled(bool enableCrashDumps)
     {
         SkipOn.Platform(SkipOn.PlatformValue.MacOs);
+        SkipOn.PlatformAndArchitecture(SkipOn.PlatformValue.Windows, SkipOn.ArchitectureValue.X86);
```

Comment:
Maybe we _should_ run this on x86 too, and ensure that nothing happens? 🤔 Alternatively, create a separate test that only runs on x86 and ensures there's no issues

Reply:
I guess we can run that one